### PR TITLE
Added Input Parameters needed for WaaS

### PR DESCRIPTION
**Pull request by shishir-msft**

### CODE REVIEW

File: azuredeploy.json [hunk 1]
```diff
@@ -359,6 +359,24 @@
...
+            "defaultValue": "Premium_LRS",
+            "allowedValues": [
+                "Premium_LRS",
+                "Standard_LRS"
+            ],
+            "metadata": {
+                "description": "Azure storage type for NFS Disks created on controller VM. Applicable only when parameter 'fileServerType' has value 'nfs'."
+            }
+        },
+        "nfsOnCtlrVmDiskSizeGB": {
```
Comment (venu-sivanadham) [inline azuredeploy.json:373]: fileServerDiskSize parameter already exists for this purpose

File: azuredeploy.json [hunk 2]
```diff
@@ -359,6 +359,24 @@
             },
             "type": "string"
         },
+        "nfsOnCtlrVmDiskStorageType": {
```
Comment (venu-sivanadham) [inline azuredeploy.json:362]: We can name it as fileServerDiskType

**Review by venu-sivanadham — CHANGES_REQUESTED** [review]

File: azuredeploy.json [hunk 3]
```diff
@@ -2,9 +2,9 @@
     "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "TypeOfTemplate":{
-            "defaultValue":"LAMP ARM TEMPLATE",
-            "type":"string"
+        "TypeOfTemplate": {
```
[the remark this answers is not in the document]
Comment (shishir-msft) [reply azuredeploy.json:5]: This is an existing parameter. Hence not removing as part of this PR.

File: azuredeploy.json [hunk 4]
```diff
@@ -576,6 +594,175 @@
...
+        },
+        "dbUsername": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "Name of the user to be used for DB."
+            }
+        },
+        "dbLoginPassword": {
+            "type": "string",
```
[the remark this answers is not in the document]
Comment (shishir-msft) [reply azuredeploy.json]: Done, thanks.

File: azuredeploy.json (hunk 1 above)
Comment (shishir-msft) [reply azuredeploy.json]: Done, thanks.

**Review by venu-sivanadham — APPROVED** [review]

**Pull request merged**